dependencies.list

```diff
@@ -1,2 +1,2 @@
-REALM_CORE_VERSION=2.8.1
-REALM_SYNC_VERSION=1.10.0
+REALM_CORE_VERSION=2.8.4
+REALM_SYNC_VERSION=2.0.0-rc9
```

Comment:
Ideally we'd have CI test with both 1.x and 2.x, but we don't have a good way to express that via dependencies.list. While we're testing only one of the two options, I don't think there's a good reason to prefer testing 1.x to 2.x.